pulp_container/app/serializers.py

```diff
@@ -309,10 +325,16 @@ class ContainerDistributionSerializer(DistributionSerializer):
     repository_version = RepositoryVersionRelatedField(
         required=False, help_text=_("RepositoryVersion to be served"), allow_null=True
     )
+    remote = DetailRelatedField(
+        required=False,
+        help_text=_("Remote that can be used to fetch content when using pull-through caching."),
+        view_name_pattern=r"remotes(-.*/.*)?-detail",
+        queryset=models.ContainerRemote.objects.all(),
+    )
```

Comment:
Should this be limited to pull-through remotes?
Is this even applicable to standard container distributions?

Reply:
This remote is a standard remote we are aware of in the pulp_container ecosystem. Under the hood, this remote contains `remote.url = pull_through_remote.url` and `remote.upstream_name = requested_upstream_name`. The remote is attached to a newly created distribution and it is used for pull-through cache downloading (https://github.com/pulp/pulp_container/blob/f6e057f3ae5c44868155cb5657daf072b24f9ec4/pulp_container/app/registry_api.py#L1569).

Reply:
Regarding the testing, this is limited to the distributions/remotes created during the initialization of pull-through caching entities in `get_pull_through_drv::if not root_cache_distribution`. Thus, we do not need to worry about standard distributions downloading remote data from attached remotes.